Deploy/stacks/Hamburg/stack-manager-inputs/inputs/data/hh-visualisation/index.html

```diff
@@ -0,0 +1,256 @@
+        var manager = new Manager(MapProvider.MAPBOX);
+        var relativePath;
+        if (window.location.href.includes("visualisation")) {
+            relativePath = "..";
+        } else {
+            relativePath = ".";
+        }
+
+        var dispersionHandler = new DispersionHandler(relativePath, manager);
+
+        var dispersionPromise = dispersionHandler.setDispersions(dispersionHandler);
+        // Read credentials
+        var credentialPromise = manager.readCredentials();
+        // Only start the map after data definitions have been read.
+        var dataPromise = manager.loadDefinitions();
+
+        // Only start the map after data definitions have been read.
+        Promise.all([dispersionPromise, credentialPromise, dataPromise]).then(() => start());
+
+        /**
+         * Starts the visualisation setup process.
+         */
+        function start() {
+            // Initialise the map object
+            manager.initialiseMap();
+
+            // ===== CUSTOMISABLE =====
+            // Set the default content in the "General" tab of the side panel
+            manager.getPanelHandler().setTitle("<h1>Digital twin of Hamburg</h1>");
+            manager.getPanelHandler().setContent(`
+            <p>This interface shows Hamburg represented in The World Avatar. Visualised applications include:</p>
+                    <b>1) Administrative limits </b><br>
+                    <b>2) Schools </b><br>
+                    <b>3) Visualisation of pollutant plumes from ships </b><br>
+                    The plumes are generated from a software called AERMOD using weather and ship data.
+                    <br><br>
+                </p>
+                `);
+
+            manager.getPanelHandler().setFooter("The World Avatar, " + new Date().getFullYear());
+            // ===== CUSTOMISABLE =====
+
+            // Save general tab state as default
+            manager.getPanelHandler().storeDefault();
+
+            // Once the underlying style has loaded...
+            MapHandler.MAP.on("style.load", function () {
+
+                // Load registered images and linked files
+                manager.loadImagesAndLinks().then(() => {
+                    manager.plotData();
+                }).then(() => {
+                    return dispersionHandler.buildDropdown(dispersionHandler);
+                }).then(() => dispersionHandler.plotData());
+            });
+
+            // Set customised legend in panel
+            manager.getPanelHandler().setLegend(`
+                    <div id='legend-container'>
+                    </div>
+                `);
+
+            // MapHandler.MAP.doubleClickZoom.disable();
+            // MapHandler.MAP.on("dblclick", (e) => {
+            //     dispersionHandler.createVirtualSensor(e.lngLat.lat, e.lngLat.lng).done().then(() => dispersionHandler.plotData());
+            // });
+
+            // dispersionHandler.updateVirtualSensors();
```

Comment:
Could this commented-out code still be useful? If so, please add a comment what it could be useful for.

Reply:
Previously uncommented to prevent unwanted addition of virtual sensors during demo